Dockerfiles/gophermart/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# syntax=docker/dockerfile:1
+
+# Build the application from source
+FROM golang:1.20
+
+WORKDIR /app
+
+COPY ./go.mod ./go.mod
+COPY ./go.sum ./go.sum
+RUN go mod download
+
+COPY ./bin ./bin
+COPY ./cmd/gophermart ./cmd/gophermart
+COPY ./db/gophermart ./db/gophermart
+COPY ./docs/gophermart ./docs/gophermart
```

Comment:
Why do you need docs in containers?

Reply:
for swagger.
Swagger is already available for auth service by address `http://ip-krakenD:port-krakenD/api/user/swagger/index.html`
<img width="926" alt="image" src="https://github.com/kripsy/gophermart/assets/3986620/8beb594d-c951-4736-8135-a70244b07da7">